### PR TITLE
Backend Updates

### PR DESCRIPTION
**Pull request by jebidiah252**
This is the current update of the backend.

### CODE REVIEW

File: backend/api/user.go [hunk 1]
```diff
@@ -3,17 +3,54 @@ package api
...
+	}
+
+	u := *new(models.User)
+	if err = c.Bind(&u); err != nil {
+		return err
+	}
+	u.SetUUID()
+
+	if err := db.CreateUser(u); err != nil {
+		log.Error("An error occurred while getting game records: %v", err)
```
Comment (rebeccamacias) [inline backend/api/user.go]: This needs to be changed to something like "An error occurred while creating a new user"

File: backend/api/user.go [hunk 2]
```diff
@@ -36,6 +73,7 @@ func returnAllUsers(c echo.Context) error {
 func init() {
 	log.Info("Running user init")
 
+	registerRoute(route{method: http.MethodGet, path: "/user/:id", handler: user})
```
Comment (jkwhite91) [inline backend/api/user.go]: I believe the handler for this should be the GetUserByID function rather than user.

File: backend/api/user.go (hunk 1 above)
Comment (mpt777) [reply backend/api/user.go]: Resolved in new push,

File: backend/api/user.go (hunk 2 above)
Comment (mpt777) [reply backend/api/user.go]: Resolved in new push,

**Comment by lynnbridge** [comment]
Can you fix these merge conflicts?

**Comment by jebidiah252** [comment]
> 

Not without write access

**Pull request merged**